conf/modules.config

```diff
@@ -691,13 +781,86 @@ process{
 
 // PAIR_VARIANT_CALLING
 
+    //ASCAT
+    withName: 'ASCAT' {
+        ext.args = {
+            [
+            "gender": meta.sex,
+            //"genomeVersion": "hg19"
+            //"purity": (optional),
+            //"ploidy": params.ploidy,
+            //"gc_files": (optional),
+            //"minCounts": (optional),
+            //"chrom_names": (optional),
+            //"min_base_qual": (optional),
+            //"min_map_qual": (optional),
+            //"ref_fasta": (optional),
+            //"skip_allele_counting_tumour": (optional),
+            //"skip_allele_counting_normal": (optional)
+            ]
+        }
+    }
+
+    //CONTROLFREEC
+    withName: 'MPILEUP_NORMAL' {
+        ext.prefix = { "${meta.id}.normal" }
+    }
+
+    withName: 'MPILEUP_TUMOR' {
+        ext.prefix = { "${meta.id}.tumor" }
+    }
+
+    withName: 'CAT_MPILEUP_NORMAL' {
+        ext.prefix = { "${meta.id}.normal.pileup" }
+    }
+
+    withName: 'CAT_MPILEUP_TUMOR' {
+        ext.prefix = { "${meta.id}.tumor.pileup" }
+    }
+
+    withName: 'FREEC_SOMATIC'{
+        ext.args = {[
+                    "sample":[
+                        inputformat: 'pileup',
+                        mateorientation: 'FR'
+                    ],
+                    "general" :[
+                        bedgraphoutput: "TRUE",
+                        breakpointthreshold: params.wes ? "1.2" : "0.8", //Values taken from Freec example configs
+                        breakpointtype: params.wes ? "4" : "2", // Values taken from Freec example configs
+                        coefficientofvariation: params.cf_coeff,
+                        contamination: params.cf_contamination ?: "",
+                        contaminationadjustment: params.cf_contamination_adjustment ? "TRUE" : "",
+                        forcegccontentnormalization: params.wes ? "1" : "0",
+                        minimalsubclonepresence: params.wes ? "30" : "20",
+                        noisydata: params.wes ? "TRUE" : "FALSE",
+                        ploidy: params.ploidy,
+                        printNA:  params.wes ? "FALSE" : "TRUE",
+                        readcountthreshold:  params.wes ? "50" : "10",
+                        sex: meta.gender,
+                        //uniquematch: not set
+                        window: params.cf_window ?: ""
+                        ],
+                    "BAF":[
+                        minimalcoverageperposition: params.cf_mincov ?: "",
+                        minimalqualityperposition: params.cf_minqual ?: "",
+                        //"shiftinquality": (optional)not set
+                    ],
+                    "control":[
+                        inputformat: "pileup",
+                        mateorientation: "FR"
+                    ]
+                ]
+            }
+    }
```

Comment:
```suggestion
    withName: 'FREEC_SOMATIC'{
        ext.args = {[
                    "sample":[
                        inputformat: 'pileup',
                        mateorientation: 'FR'
                    ],
                    "control":[
                        inputformat: "pileup",
                        mateorientation: "FR"
                    ],
                    "general" :[
                        bedgraphoutput: "TRUE",
                        breakpointthreshold: params.wes ? "1.2" : "0.8", //Values taken from Freec example configs
                        breakpointtype: params.wes ? "4" : "2", // Values taken from Freec example configs
                        coefficientofvariation: params.cf_coeff,
                        contamination: params.cf_contamination ?: "",
                        contaminationadjustment: params.cf_contamination_adjustment ? "TRUE" : "",
                        forcegccontentnormalization: params.wes ? "1" : "0",
                        minimalsubclonepresence: params.wes ? "30" : "20",
                        noisydata: params.wes ? "TRUE" : "FALSE",
                        ploidy: params.ploidy,
                        printNA:  params.wes ? "FALSE" : "TRUE",
                        readcountthreshold:  params.wes ? "50" : "10",
                        sex: meta.gender,
                        //uniquematch: not set
                        window: params.cf_window ?: ""
                        ],
                    "BAF":[
                        minimalcoverageperposition: params.cf_mincov ?: "",
                        minimalqualityperposition: params.cf_minqual ?: "",
                        //"shiftinquality": (optional)not set
                    ]
                ]
            }
    }
```

What do you think about having the `control` section this way?

Reply:
seems sensible

Reply:
I'll commit your changes in a second.